funnel/views/participant.py

```diff
@@ -173,6 +173,21 @@ def participant_badge(profile, project, participant):
     )
 
 
+@app.route('/<profile>/<project>/participant/<participant_id>/label_badge')
+@funnelapp.route('/<project>/participant/<participant_id>/badge', subdomain='<profile>')
+@lastuser.requires_login
+@load_models(
+    (Profile, {'name': 'profile'}, 'g.profile'),
+    ((Project, ProjectRedirect), {'name': 'project', 'profile': 'profile'}, 'project'),
+    (Participant, {'id': 'participant_id'}, 'participant'),
+    permission='checkin_event',
+)
```

Comment:
@iambibhas Use of `load_models` should be deprecated in favour of `ModelView`. Applies to the other views in this file as well.

Reply:
Keeping this for a separate PR. All the views on this file need to be moved to ModelView.